docs/how_keyboards_work.md

```diff
@@ -33,7 +33,11 @@ The firmware does not send actual letters or characters, but only scancodes.
 Thus, by modifying the firmware, you can only modify what scancode is sent over
 USB for a given key.
 
-## 3. What the Operating System Does
+## 3. What the Event Input/Kernel Does
+
+The *scancode* is mapped to a *keycode* dependant on the keyboard [60-keyboard.hwdb at Master](example: https://github.com/systemd/systemd/blob/master/hwdb/60-keyboard.hwdb ). Without this mapping, the operating system will not receive a valid keycode and will be unable to do anything useful with that key press.
```

Comment:
Links like this need to be formatted like `[text](https://link.com), otherwise it won't get picked-up as a valid url :)

Reply:
```suggestion
The *scancode* is mapped to a *keycode* dependant on the keyboard [60-keyboard.hwdb at Master](https://github.com/systemd/systemd/blob/master/hwdb/60-keyboard.hwdb). Without this mapping, the operating system will not receive a valid keycode and will be unable to do anything useful with that key press.
```